### ISSUE TITLE
An issue about the weight calculation

### ISSUE THREAD

**Issue by hust0zzcc** [vcg-uvic/learned-correspondence-release#2]
Thanks for releasing your code!
The weight of each correspondence is computed by: 'weights = tf.nn.relu(tf.tanh(self.logits))' in 'network.py'. But in the Figure.2 of your paper, it should be computed by: 'weights = tf.tanh(tf.nn.relu(self.logits))'. Could you please tell me which one is better?
Thanks.

**Comment by kmyid** [comment]
The one on the code is correct. However, they should be identical actually.
